lib/curl/curl.nit

```diff
@@ -450,15 +503,18 @@ class HeaderMap
 		return col
 	end
 
-	fun length: Int do return arr.length
-	fun is_empty: Bool do return arr.is_empty
+	# Number of values in `self`
+	fun length: Int do return array.length
+
+	# Is this map empty?
+	fun is_empty: Bool do return array.is_empty
 end
 
-class HeaderMapIterator
+private class HeaderMapIterator
```

Comment:
What a `HeaderMapIterator` does more than a basic `MapIterator`? If nothing I suggest to get rid of it completely